test/sql_tests.cpp

```diff
@@ -301,6 +298,174 @@ TEST(StringLengthTest) {
   ASSERT_EQ(result.getStatement(2)->stringLength, 21);
 }
 
+TEST(ExceptOperatorTest) {
```

Comment:
I think a test where the last SELECT has its own ORDER/LIMIT in addition to the global one would help avoid future problems.

Reply:
Sure